src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/configuration/KafkaConsumerConfig.java

```diff
@@ -93,6 +99,11 @@
                 new ConcurrentKafkaListenerContainerFactory<>();
         factory.setConsumerFactory(newMainConsumerFactory());
         factory.getContainerProperties().setAckMode(ContainerProperties.AckMode.MANUAL_IMMEDIATE);
+        factory.setErrorHandler((exception, data) -> {
```

Comment:
Similar to above, we should probably try to add a error handler to all 3 consumers.

Its also possible that this error handler gets called with something that is not a deserialization exception, as it stands it would log out "Failed to deserialize message"